R/build_leanpub_embed_files.R

```diff
@@ -0,0 +1,41 @@
+# script to fix embedded files for Leanpub
+# originally used the raw rmd files before conversion  with bookdown_to_leanpub() - now part of the function at the beginning
+
+library(here)
```

Comment:
I think since you added these in the dependencies list, having them called by `library()` here is not necessary.